archive/untar.ts

```diff
@@ -86,7 +86,6 @@ function parseHeader(buffer: Uint8Array): TarHeader {
 }
 
 /** Tar entry */
-// deno-lint-ignore no-empty-interface
```

Comment:
Canary complains about this, but stable wants this warning. Since our CI lints only with the canary release I had to remove this to get it green.